manifests/enterprise.pp

```diff
@@ -28,6 +28,8 @@
 #
 # @param heap_size Value of the HEAP_SIZE environment variable.
 #
+# @param max_open_files Value of the MAX_OPEN_FILES environment variable.
```

Comment:
Why is the parameter added to three different classes?

Reply:
I followed the current approach for similar cases, but actually you're right, max_open_files in this case can and should not stay in package.pp. Actually if we split the sensu.erb template used both by /etc/default/sensu and /etc/default/sensu-enterprise we will be able to set ee-only parameters only in enterprise.pp

Reply:
Do we need it anywhere except the enterprise class then?

Reply:
Actually, we need these params also in init.pp, besides enterprise.pp for how the module is done and tested (all spec tests for the various subclasses, in reality, test on the main class)

Reply:
Do we need it in the package class then?

Reply:
Parameters used only in puppet enterprise are not needed in packages class.
FYI just pushed with the code to support other sense ee settings like JAVA_OPTS.
Have to properly test on vagrant yet. 
These changes anyway, might be obsoleted by what we decide in #851 .